dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -659,17 +661,113 @@
             <SolidColorBrush x:Key="CircleElevationBorderBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
             <SolidColorBrush x:Key="AccentControlElevationBorderBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
 
-            <!-- System high contrast colors -->
+            <!-- Colors copied from Default theme to match list of keys -->
```

Comment:
Since these colors should not ever be used in HC I assign all of them a value of `#FF0000` red so it is obvious feedback to devs they have something hooked up wrong when visual testing. @YuliKl what do you think?

Reply:
Updated.